docs: refine the step by step docs (#31)
pingcap/pingcap.github.io: docs/setup/database.md

# Create TiDB Serverless cluster

1. Click [here](https://tidbcloud.com/console/clusters/create-cluster?utm_source=github&utm_medium=ossinsight_lite) to create a TiDB Cloud Serverless cluster.

   <div align="center">
      <img src="images/create-cluster.png" width="1000px" alt="Create TiDB Serverless cluster page"/>
      <p><i>Create TiDB Serverless cluster page</i></p>
   </div>

> 2. After waiting for the cluster to be created (About a few seconds), click the **Connect** button in the upper right corner to open the **Connect to** dialog.

Author: ```suggestion
2. When the cluster creation is complete (about a few seconds), click the **Connect** button in the upper right corner to open the **Connect to** dialog.
```
